gatewayconfig/bluetooth/services/helium_service.py

```diff
@@ -7,8 +7,8 @@
 from gatewayconfig.bluetooth.characteristics.mac_address_characteristic import MacAddressCharacteristic
 from gatewayconfig.bluetooth.characteristics.lights_characteristic import LightsCharacteristic
 from gatewayconfig.bluetooth.characteristics.wifi_ssid_characteristic import WifiSSIDCharacteristic
-from gatewayconfig.bluetooth.characteristics.assert_location_characteristic import AssertLocationCharacteristic
-from gatewayconfig.bluetooth.characteristics.add_gateway_characteristic import AddGatewayCharacteristic
+# from gatewayconfig.bluetooth.characteristics.assert_location_characteristic import AssertLocationCharacteristic
```

Comment:
this will be come part of wallet management app or dashboard.

Reply:
@pritamghanghas yeah I think @vpetersson doesn't like leaving in commented code 😂

But yeah if we don't need assert London we should probably remove it.

What I don't understand is the this is done in the app now as it surely needs to be signed by the Helium wallet but perhaps that is done by a deep link?